docs/particles/ionization_states.rst

```diff
@@ -0,0 +1,141 @@
+.. _ionization-state-data-structures:
+
+Ionization state data structures
+********************************
+
+The ionization state (or charge state) of a plasma refers to the
+fraction of an element that is at each ionization level.  For example,
+the ionization state of a pure helium plasma could be 5% He⁰⁺, 94% He¹⁺,
+and 1% He²⁺.
+
+The ionization state of a single element
+========================================
+
+We may use the `~plasmapy.particles.IonizationState` class
+to represent the ionization state of a single element, such as for this
+example.
+
+>>> from plasmapy.particles import IonizationState
+>>> ionization_state = IonizationState("He", [0.05, 0.94, 0.01])
+
+The ionization state for helium may be accessed using the
+``ionic_fractions`` attribute.  These ionic fractions correspond to the
+``integer_charges`` attribute.
+
+>>> ionization_state.ionic_fractions
+array([0.05, 0.94, 0.01])
+>>> ionization_state.integer_charges
+array([0, 1, 2])
+
+The ``Z_mean`` attribute returns the mean integer charge averaged
+over all particles in that element.
+
+>>> ionization_state.Z_mean
+0.96
+
+The ``Z_rms`` attribute returns the root mean square integer charge.
+
+>>> ionization_state.Z_rms
+0.9899...
+
+The ``Z_most_abundant`` attribute returns a `list` of the most abundant
+ion(s).  The `list` may contain more than one integer charge in case of
+a tie.
+
+>>> ionization_state.Z_most_abundant
+[1]
+
+The ``info`` method prints out the ionic fraction for the ions with
+an abundance of at least 1%.
+
+>>> ionization_state.info()
+IonizationState instance for He with Z_mean = 0.96
+----------------------------------------------------------------
+He  0+: 0.050
+He  1+: 0.940
+He  2+: 0.010
+----------------------------------------------------------------
```

Comment:
I wonder if we could get more descriptive than `info()`. `summarize()`? `summary` as a property? `describe()`, following pandas?

Reply:
> `summarize()`?

I was thinking the same thing!  I like both `summarize()` and `describe()`, with a slight preference for `summarize()`.